### PR TITLE
Add dependency update check plugin + bump dependencies

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -2,10 +2,12 @@ plugins {
   id 'org.gradle.checkstyle'
   id 'org.gradle.java-library'
 
-  id 'com.diffplug.gradle.spotless' version '3.29.0' apply false
-  id 'io.freefair.lombok' version '5.0.1' apply false
-  id 'org.ajoberstar.grgit' version '4.0.2' apply false
-  id 'org.checkerframework' version '0.5.1' apply false
+  id 'com.github.ben-manes.versions' version '0.28.0'
```
Comment (mkondratek) [inline build.gradle:5]: How does it work?

Comment (PawelLipski) [reply build.gradle:5]: Done, added docs

**Comment by PawelLipski** [comment]
@mkondratek hmm I couldn't reproduce the issue locally... could you show your `ls -l ~/.gradle/wrapper/dists`?

**Comment by PawelLipski** [comment]
Also, did you perform `runIde` from IntelliJ or via `./gradlew runIde`?

**Pull request merged**